independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html

```diff
@@ -35,140 +56,239 @@
     </div>
-    <div class="right-column">
-        <div class="right-section">
-            <h3>Application</h3>
-            <ul>
-                <li>GroupId: <code>{project.group-id}</code></li>
-                <li>ArtifactId: <code>{project.artifact-id}</code></li>
-                <li>Version: <code>{project.version}</code></li>
-                <li>Quarkus Version: <code>{quarkus.version}</code></li>
-            </ul>
-        </div>
-        <div class="right-section">
-            <h3>Do you like Quarkus?</h3>
-            <ul>
-                <li>Go give it a star on <a href="https://github.com/quarkusio/quarkus">GitHub</a>.</li>
-            </ul>
+</div>
+
+<div class="banner-wrapper">
+    <div class="grid">
+        <div class="banner">
+            <div class="grid">
+                <div class="callout">Congratulations!</div>
+                <div class="appdetails">
+                    <h5>Application</h5>
+                    <ul>
+                        <li>GroupId: <code>{project.group-id}</code></li>
+                        <li>ArtifactId: <code>{project.artifact-id}</code></li>
+                        <li>Version: <code>{project.version}</code></li>
+                        <li>Quarkus Version: <code>{quarkus.version}</code></li>
+                    </ul>
+                </div>
+            </div>
         </div>
-        {#if input.selected-extensions}
-        <div class="right-section">
-            <h3>Selected extensions guides</h3>
-            <ul>
-                {#for ext in input.selected-extensions}
-                {#if ext.guide}
-                <li title="{ext.description}"><a href="{ext.guide}" target="_blank">{ext.name} guide</a></li>
-                {/if}
-                {/for}
-            </ul>
+    </div>
+</div>
+
+<div class="content-wrapper">
+    <div class="grid">
+        <div class="left-column">
+            <h1>You just made a Quarkus application.</h1>
+            <h4>Try running Quarkus Dev and <a href="/q/dev/">Dev UI</a> to view your installed extensions.</h4>
+            <p>This page is served by Quarkus. The source is in <code>src/main/resources/{config.file-name}</code>.</p>
```

Comment:
The `config.file-name` variable doesn't look right for this item.

Reply:
I was copying it from the previous page. I have no idea what it should be.